### PR TITLE
adds gpses to literally every ghostrole ship + sfa powerloader

### PR DESCRIPTION
**Pull request by RustingWithYou**
Most ghost role ships now have GPSes, to make exoplanet navigation easier.

The SFA now has a new variant of the solarian powerloader, missing one of its clamps due to damage. In addition, the power loader aboard the FSF corvette has been replaced with a solarian power loader.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: code/modules/heavy_vehicle/premade/powerloader.dm [hunk 1]
```diff
@@ -25,6 +25,10 @@
 	name = "solarian powerloader"
 	e_color = COLOR_DARK_GREEN_GRAY
 
+/mob/living/heavy_vehicle/premade/ripley/loader/sol/fucked
```
Comment (SleepyGemmy) [inline code/modules/heavy_vehicle/premade/powerloader.dm:28]:
```suggestion
/mob/living/heavy_vehicle/premade/ripley/loader/sol/damaged
```
non-descriptive path.

File: html/changelogs/rustingwithyou - offshipgps.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "all ghost ships now have GPS units"
```
Comment (SleepyGemmy) [inline html/changelogs/rustingwithyou - offshipgps.yml:41]:
```suggestion
  - maptweak: "All ghost ships now have GPSes."
```

File: html/changelogs/rustingwithyou - offshipgps.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  - maptweak: "adds a amaged solarian powerloader and adds it to the sfa ship"
```
Comment (SleepyGemmy) [inline html/changelogs/rustingwithyou - offshipgps.yml:42]:
```suggestion
  - maptweak: "Adds a damaged Solarian powerloader and adds it to the SFA away ship."
```

File: maps/away/ships/sol_pirate/sfa_patrol_ship.dmm [hunk 4]
```diff
@@ -1873,6 +1874,7 @@
 	pixel_x = 24;
 	req_access = list(113)
 	},
+/mob/living/heavy_vehicle/premade/ripley/loader/sol/fucked,
```
Comment (SleepyGemmy) [inline maps/away/ships/sol_pirate/sfa_patrol_ship.dmm:1877]:
```suggestion
/mob/living/heavy_vehicle/premade/ripley/loader/sol/damaged,
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: html/changelogs/rustingwithyou - offshipgps.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  - maptweak: "replaces the FSF corvette's powerloader with a solarian powerloader"
```
Comment (SleepyGemmy) [inline html/changelogs/rustingwithyou - offshipgps.yml:43]:
```suggestion
  - maptweak: "Replaces the FSF corvette's powerloader with a Solarian powerloader."
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**